src/sentry/static/sentry/app/components/dropdownMenu.jsx

```diff
@@ -363,6 +363,10 @@ class DropdownMenu extends React.Component {
       getRootProps: this.getRootProps,
       getActorProps: this.getActorProps,
       getMenuProps: this.getMenuProps,
+      actions: {
+        open: this.handleOpen,
+        close: this.handleClose,
```

Comment:
When would you need to control the menu state this way?

Reply:
Oops sorry, forgot to link the two